diktat-rules/src/test/resources/test/paragraph2/kdoc/package/src/main/kotlin/com/saveourtool/diktat/kdoc/methods/KdocWithoutThrowsTagExpected.kt

```diff
@@ -0,0 +1,17 @@
+package test.paragraph2.kdoc.`package`.src.main.kotlin.com.saveourtool.diktat.kdoc.methods
+
+/**
+ * @param onSuccess
+ * @param onFailure
+ */
+fun parseInputNumber(onSuccess: (number: Int) -> Unit, onFailure: () -> Unit) {
+    try {
+        val input: Int = binding.inputEditText.text.toString().toInt()
+        if (input < 0)
+            throw NumberFormatException()
+
+        onSuccess(input)
+    } catch (e: NumberFormatException) {
+        onFailure()
+    }
+}
```

Comment:
I think you can use only warnings tests, no need in fix tests. Also, please add more tests. For example, for case when the thrown exception is child of exception described in catch block. Or when catch block is empty.

Reply:
Done